### PR TITLE
MDS textarea cell

### PR DESCRIPTION
**Pull request by juanca**
https://www.pivotaltracker.com/story/show/180827609

## PR upkeep checklist

- [x] Change log entry
- [x] Label(s)
- [x] Assignee(s)
- [x] Deployment URL: https://mavenlink.github.io/design-system/mds-textarea/
- [x] (When ready for review) Reviewer(s)
- [x] Green Bigmaven CI check https://github.com/mavenlink/mavenlink/pull/27630
- [ ] DevQA on Bigmaven staging (e.g. does the work need to be imported in the internal design system?)

### CODE REVIEW

**Comment by juanca** [comment]
- [ ] css pollution: min-height
- [ ] css missing: no scrollbars

**Comment by juanca** [comment]
API

- [ ] classNames.textarea
- [ ] classNames.container
- [ ] onChange
- [ ] ref.value

File: src/components/cell-control/textarea.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  composes: container from './cell-control.css';
+  composes: container from '../control/input.css';
+  position: relative;
+}
+
+.textarea {
+  composes: input from '../control/input.css';
+  border: none;
+  height: 100%;
+  min-height: unset !important;
```
Comment (Bloodyaugust) [inline src/components/cell-control/textarea.css:13]: 😭

Comment (Bloodyaugust) [reply src/components/cell-control/textarea.css:13]: If only `composes` guaranteed order

File: src/components/cell-control/textarea.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+import Tooltip from '../tooltip/tooltip.jsx';
+import styles from './textarea.css';
+import useValidation from '../../hooks/use-validation.jsx';
+
+const Textarea = forwardRef(function Textarea(props, ref) {
+  const ids = {
+    invalidIcon: `${props.id}-invalid-icon`,
+    invalidTooltip: `${props.id}-invalid-tooltip`,
+  };
+  const refs = {
```
Comment (Bloodyaugust) [inline src/components/cell-control/textarea.jsx:15]: Does this need to be a dictionary, since it has only one entry?

**Review by Bloodyaugust — APPROVED** [review]
LGTM! One minor non-blocking suggestion

File: src/components/cell-control/textarea.css (hunk 1 above)
Comment (juanca) [reply src/components/cell-control/textarea.css:13]: The min-height is css pollution 😭

File: src/components/cell-control/textarea.jsx (hunk 2 above)
Comment (juanca) [reply src/components/cell-control/textarea.jsx:15]: 🤷  No but I see it as a useful pattern for MDS components. We have between 0, 1, or many refs. Setting up the structure now makes it easier for the next dev? Only time will tell! ⏳

**Pull request merged**